nix/modules/fedimintd.nix

```diff
@@ -110,11 +141,6 @@ let
           and is set to be read only.
         '';
       };
-      rustLogEnv = mkOption {
-        type = types.str;
-        default = "info,fedimint_server::request=debug,fedimint_client::request=debug";
-        description = "Value to set RUST_LOG to";
-      };
```

Comment:
Should we make this a default env var?

Reply:
I kind of think that default should be the default (which is `info`). Might be an opportunity to tweak the logging statements itself to be better at default level.